.github/workflows/ci.yml

```diff
@@ -42,6 +42,12 @@ jobs:
       - name: Run cargo check (without dev-dependencies to catch missing feature flags)
         if: startsWith(matrix.rust, 'nightly')
         run: cargo check -Z features=dev_dep
+      - name: Add rust-src
+        if: startsWith(matrix.rust, 'nightly')
+        run: rustup component add rust-src
+#      - name: Check selected Tier 3 targets
+#        if: startsWith(matrix.rust, 'nightly') && matrix.os == 'ubuntu-latest'
+#        run: cargo check -Z build-std --target=riscv32imc-esp-espidf
```

Comment:
Why is this commented out?

Reply:
See this: https://github.com/smol-rs/async-io/pull/144#issuecomment-1666927490

Reply:
Ah, thanks for the explanation. Could you add a todo comment with the explanation on it or a link to your comment?

Reply:
Done.